src/api/getPortalUriWithToken.ts

```diff
@@ -32,8 +34,8 @@ export async function getPortalUriWithToken(
 		const response = await makeRESTRequest(
 			"POST",
 			spec,
-			{ apiVersion: 1, namespace: "%SYS", path: "/action/query" },
-			{ query: "select %Atelier_v1_Utils.General_GetCSPToken(?, ?) token", parameters: [PORTAL_HOME, token] },
+			{ apiVersion: 1, namespace, path: "/action/query" },
```

Comment:
should this be: `namespace: namespace` following the syntax in the deleted line?

Reply:
That's implicit in TypeScript